modules/nextflow/src/main/groovy/nextflow/executor/GridTaskHandler.groovy

```diff
@@ -100,6 +102,27 @@ class GridTaskHandler extends TaskHandler implements FusionAwareTask {
         this.sanityCheckInterval = duration
     }
 
+    @Override
+    void prepareLauncher() {
+        // -- create the wrapper script
+        createTaskWrapper(task).build()
+    }
+
+    @Override
+    String getWorkDir() {
+        fusionEnabled()
+            ? FusionHelper.toContainerMount(task.workDir).toString()
+            : task.workDir.toString()
+    }
+
+    @Override
+    List<String> getLaunchCommand() {
+        final workDir = Escape.path(getWorkDir())
+        final cmd = "bash ${workDir}/${TaskRun.CMD_RUN} 2>&1 | tee ${workDir}/${TaskRun.CMD_LOG}"
```

Comment:
Jordi was mentioning `| tee ${workDir}/${TaskRun.CMD_LOG}"` should not be done, because fusion is taking care of it. @bentsherman can you please add it only for non-fusion runs?

Reply:
Solved by https://github.com/nextflow-io/nextflow/pull/4973